app/src/app-state-reducer.js

```diff
@@ -12,10 +12,16 @@ export default function reducer(state) {
     }
   }
 
-  const { proposals } = state
+  const { proposals, stakeToken } = state
   return {
     ...state,
 
+    stakeToken: {
```

Comment:
should we also add here all the amounts that are not being converted to BN on the script? like this one https://github.com/1Hive/conviction-voting-app/blob/f34d9acb98d7ad1b7af1d0da77bf3af0c2e0246d/app/src/script.js#L280 ?

Reply:
Yes makes sense!

Reply:
yeah makes sense